delfin/tests/unit/drivers/fujitsu/eternus/test_eternus_stor.py

```diff
@@ -1131,4 +1530,52 @@ def test_list_ports_old(self):
     def test_parse_alert(self):
         parse_alert = self.driver.parse_alert(context, PARSE_ALERT_INFO)
         self.assertEqual(parse_alert.get('alert_id'), PARSE_ALERT_INFO.get(
-            '1.3.6.1.4.1.211.1.21.1.150.1.1'))
+            '1.3.6.1.2.1.1.3.0'))
+
+    def test_list_storage_host_initiators(self):
+        EternusSSHPool.get = mock.Mock(return_value={paramiko.SSHClient()})
+        EternusSSHPool.do_exec_shell = mock.Mock(
+            side_effect=[HOST_STATUS_INFO, FC_HOSTS_INFO, ISCSI_HOST_INFO,
+                         ISCSI_HOST_DETAIL_ZERO, ISCSI_HOST_DETAIL_ONE,
+                         ISCSI_HOST_DETAIL_TWO, SAS_HOST_INFO])
+        initiators = self.driver.list_storage_host_initiators(ctx)
+        self.assertEqual(initiators[0].get('native_storage_host_id'),
```

Comment:
The assert should compare a complete piece of data rather than a field value. Otherwise, the unit test is useless.
![image](https://user-images.githubusercontent.com/59107077/153791873-da58494f-5a0b-45bc-af06-3f515f346b3a.png)

Reply:
Has been modified